nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/commands/SetLogAttributes.java

```diff
@@ -139,7 +139,7 @@
         "com.sun.enterprise.server.logging.GFFileHandler.excludeFields",
         "com.sun.enterprise.server.logging.GFFileHandler.multiLineMode",
         "com.sun.enterprise.server.logging.GFFileHandler.compressOnRotation",
-        "com.sun.enterprise.server.logging.GFFileHandler.redirectSystemStreams",
+        "com.sun.enterprise.server.logging.GFFileHandler.logStandardStreams",
```

Comment:
The domain.xml property can keep the new changed name, but the logging property should have the same name, to make sure domain upgrades work

Reply:
Changing the property name will not affect the domain upgrade because the property was not added to the `logging.properties` file when it was initially implemented.